src/main/kotlin/edu/byu/mealplanningassistant/MealPlanningAssistantApplication.kt

```diff
@@ -36,7 +34,8 @@ class RecipeResource(val service: RecipeService) {
 		return service.clear()
 	}
 
-	@GetMapping("/meal-plan")
+	@CrossOrigin(origins = ["http://localhost:3000"])
```

Comment:
What does this do?

Reply:
explicitly allows the front end to communicate with the backend
Think of it as allow connections from: ___